ote/src/cljs/ote/app/controller/front_page.cljs

```diff
@@ -85,35 +86,52 @@
   (process-event [_ app]
     (dissoc app :navigation-prompt-open?))
 
-  OpenUserMenu
+  OpenServiceInfoMenu
   (process-event [_ app]
     (-> app
-      (assoc-in [:ote-service-flags :user-menu-open]
-                (if (get-in app [:ote-service-flags :user-menu-open]) false true))
+        (assoc-in [:ote-service-flags :service-info-menu-open]
+                  (if (get-in app [:ote-service-flags :service-info-menu-open]) false true))
+        (assoc-in [:ote-service-flags :my-services-menu-open] false)
+        (assoc-in [:ote-service-flags :user-menu-open] false)
+        (assoc-in [:ote-service-flags :header-open] false)))
+
+  OpenMyServicesMenu
+  (process-event [_ app]
+    (-> app
+      (assoc-in [:ote-service-flags :my-services-menu-open]
+                (if (get-in app [:ote-service-flags :my-services-menu-open]) false true))
+      (assoc-in [:ote-service-flags :service-info-menu-open] false)
       (assoc-in [:ote-service-flags :header-open] false)
+      (assoc-in [:ote-service-flags :user-menu-open] false)
       (assoc-in [:ote-service-flags :lang-menu-open] false)))
 
-  OpenHeader
+  OpenUserMenu
   (process-event [_ app]
     (-> app
-      (assoc-in [:ote-service-flags :header-open]
-              (if (get-in app [:ote-service-flags :header-open]) false true))
-      (assoc-in [:ote-service-flags :user-menu-open] false)
+      (assoc-in [:ote-service-flags :user-menu-open]
+                (if (get-in app [:ote-service-flags :user-menu-open]) false true))
+      (assoc-in [:ote-service-flags :service-info-menu-open] false)
+      (assoc-in [:ote-service-flags :my-services-menu-open] false)
+      (assoc-in [:ote-service-flags :header-open] false)
       (assoc-in [:ote-service-flags :lang-menu-open] false)))
 
   OpenLangMenu
   (process-event [_ app]
     (-> app
       (assoc-in [:ote-service-flags :lang-menu-open]
                 (if (get-in app [:ote-service-flags :lang-menu-open]) false true))
+      (assoc-in [:ote-service-flags :service-info-menu-open] false)
+      (assoc-in [:ote-service-flags :my-services-menu-open] false)
       (assoc-in [:ote-service-flags :user-menu-open] false)
       (assoc-in [:ote-service-flags :header-open] false)))
 
   CloseHeaderMenus
   (process-event [_ app]
     (-> app
-        (assoc-in [:ote-service-flags :lang-menu-open] false)
+        (assoc-in [:ote-service-flags :service-info-menu-open] false)
+        (assoc-in [:ote-service-flags :my-services-menu-open] false)
         (assoc-in [:ote-service-flags :user-menu-open] false)
+        (assoc-in [:ote-service-flags :lang-menu-open] false)
         (assoc-in [:ote-service-flags :header-open] false)))
 
```

Comment:
Consistent order for the `assoc-in` lines in each menu would make it easier to read.